class.json-api-endpoints.php

```diff
@@ -626,7 +626,43 @@ function cast_and_filter_item( &$return, $type, $key, $value, $types = array(),
 				'network'       => '(boolean)  Whether the plugin can only be activated network wide.',
 				'autoupdate'    => '(boolean)  Whether the plugin is auto updated',
 				'log'           => '(array:safehtml) An array of update log strings.',
-        'action_links'  => '(array) An array of action links that the plugin uses.',
+        		'action_links'  => '(array) An array of action links that the plugin uses.',
+			);
+			$return[$key] = (object) $this->cast_and_filter(
+				$value,
+				/**
+				 * Filter the documentation returned for a plugin.
+				 *
+				 * @module json-api
+				 *
+				 * @since 3.1.0
+				 *
+				 * @param array $docs Array of documentation about a plugin.
+				 */
+				apply_filters( 'wpcom_json_api_plugin_cast_and_filter', $docs ),
+				false,
+				$for_output
+			);
+			break;
+		case 'plugin_v1_2' :
```

Comment:
I think we could do
```
case 'plugin' :
case 'plugin_v1_2' :
```
and extend the case with
```
if ( === `plugin_v1_2` ) {
do something in particular
}
```

Reply:
good idea